opts/opts_test.go

```diff
@@ -179,6 +220,163 @@ func TestValidateExtraHosts(t *testing.T) {
 	}
 }
 
+func TestValidateAttach(t *testing.T) {
+	valid := []string{
+		"stdin",
+		"stdout",
+		"stderr",
```

Comment:
You should probably add some non-lower case values here to make sure we lower-case them in the ValidateAttach return value.